### ISSUE TITLE
Thread scroll position not being updated

### ISSUE THREAD

**Issue by ghost** [Adamantcheese/Kuroba#1201]
I noticed an issue where I would scroll to the bottom of a thread, open another thread, and when I go back to the first thread it would not update my position. Since I had been watching it, my position was saved to where I was before the newest posts were made. It was only this one thread, and restarting the app solved it (though I still had to scroll down again after restarting, so I think the issue is my position was never updated at all).

Tested on v4.13.6-6c39e65 on a Samsung Note 10 Plus

**Comment by Adamantcheese** [comment]
Weird. Can't reproduce though and since this is one-time thing I'm just gonna close this.

**Comment by ghost** [comment]
This is still happening.

**Comment by Astridchan** [comment]
Yeah, I can confirm this bug is still happening on recent dev builds. If it helps to narrow it down it only seems to happen if it's a thread I've posted a reply in.

**Comment by Twinov** [comment]
I believe I've found a way to reliably reproduce this on my Pixel 6 Pro running 4.13.6-223dfc5b  
If a thread `A` is opened in the app, then you switch to use another app and the Android memory killer kills Kuroba, then next time you open up `A` and any subsequent opens will be in the scroll position it was in before being killed, no matter where you scroll. Additionally, I've found that clicking Top in the app goes to that wrongly saved position instead of actually going to the thread top.  
I've found that opening the thread and killing the app by swiping up from the recents screen fixes the bugged scroll position.
